rest/buildlogger_routes_test.go

```diff
@@ -709,39 +909,38 @@ func (s *LogHandlerSuite) TestLogMetaGetByTestNameHandlerCtxErr() {
 	s.NotEqual(http.StatusOK, resp.Status())
 }
 
-func (s *LogHandlerSuite) TestLogGroupHandlerFound() {
+func (s *LogHandlerSuite) TestLogGroupByTestNameHandlerFound() {
 	for _, printTime := range []bool{true, false} {
 		opts := dbModel.LogIteratorReaderOptions{
 			PrintTime:     printTime,
 			PrintPriority: !printTime,
 		}
 
-		rh := s.rh["group"]
-		rh.(*logGroupHandler).opts.TaskID = "task_id1"
-		rh.(*logGroupHandler).opts.TestName = "test1"
-		rh.(*logGroupHandler).opts.Group = "tag1"
-		rh.(*logGroupHandler).opts.Tags = []string{"tag1"}
-		rh.(*logGroupHandler).opts.TimeRange = dbModel.TimeRange{
+		rh := s.rh["group_test_name"].Factory()
+		rh.(*logGroupByTestNameHandler).opts.TaskID = "task_id1"
```

Comment:
Same thing about type-casted variable.

Can we also split them into `t.Run` test cases?